ompi/mca/io/ompio/io_ompio_file_open.c

```diff
@@ -415,9 +415,13 @@ int mca_io_ompio_file_delete (const char *filename,
 
     ret = unlink(filename);
 
-    if (0 > ret && ENOENT != errno ) {
-	opal_output (1, "errno = %d %s\n", errno, strerror(errno));
-        return MPI_ERR_ACCESS;
+    if (0 > ret ) {
+        if ( ENOENT == errno ) {
+            return MPI_ERR_NO_SUCH_FILE;
+        } else {
+            opal_output (1, "errno = %d %s\n", errno, strerror(errno));
```

Comment:
Shouldn't this output stream be 0?

Also, I know you're just copying the existing `opal_output()` string, but can you make it a bit more descriptive so that if it actually happens, someone will have a clue as to what the errno value is being reported for?  E.g., if you just see a `errno = X` string in the middle of your MPI output, it'll be quite confusing to know what that is.